controllers/pxc/vaultConf.go

```diff
@@ -82,3 +101,11 @@ func vaultTransport(cert []byte) (http.RoundTripper, error) {
 		},
 	}, nil
 }
+
+func readVaultToken() ([]byte, error) {
+	path := defaultTokenPath
+	if envPath := os.Getenv("VAULT_TOKEN"); envPath != "" {
```

Comment:
Using `os.LookupEnv` for that is more go stile

Reply:
`VAULT_TOKEN` is an environment variable that the Vault Go API package uses: https://pkg.go.dev/github.com/hashicorp/vault/api#NewClient

Using the same variable name but with a different meaning seems like it could introduce undefined behavior.

Reply:
96fea2e